src/inject.js

```diff
@@ -64,8 +64,21 @@ const stringHandlers = {
         if (Array.isArray(val)) {
             return val.map(fontFamily).join(",");
         } else if (typeof val === "object") {
-            injectStyleOnce(val.src, "@font-face", [val], false);
-            return `"${val.fontFamily}"`;
+            let key = val.src;
+            let fontFamily = val.fontFamily;
+
+            // More than likely this object will be an OrderedElement type
+            // which will require usage of its getters methods
+            if (val instanceof OrderedElements) {
+                // We need to generate a unique key by which we can identify
+                // this declaration in the the 'alreadyInjected' object
+                key = `fontface_${hashObject(val)}`;
```

Comment:
Sorry for the late reply, was away. No, there isn't a reason I didn't use `val.get()`. Good point. I'll change it to be consistent like you said. Thanks. And I'll also add the tests.